### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by SpiderDan98**
- Add unit tests

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,10 @@
     }
   ],
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "spryker/api": "^0.2.0",
```
Comment (daniel-rose) [inline composer.json]: @SpiderDan98 did you consider all possible versions?

File: src/FondOfSpryker/Zed/CompanyUnitAddressApi/Business/Model/CompanyUnitAddressApi.php [hunk 2]
```diff
@@ -121,7 +121,7 @@ public function get(int $idCompanyUnitAddress): ApiItemTransfer
             $companyUnitAddressTransfer
         );
 
-        if ($companyUnitAddressTransfer === null) {
+        if ($companyUnitAddressTransfer->getAddress1() === null) {
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompanyUnitAddressApi/Business/Model/CompanyUnitAddressApi.php:124]: Why?

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: src/FondOfSpryker/Zed/CompanyUnitAddressApi/Business/Model/CompanyUnitAddressApi.php (hunk 2 above)
Comment (SpiderDan98) [reply src/FondOfSpryker/Zed/CompanyUnitAddressApi/Business/Model/CompanyUnitAddressApi.php:124]: @daniel-rose The method getCompanyUnitAddressById cant return null. Therefore I check now the address1 field.

File: composer.json (hunk 1 above)
Comment (SpiderDan98) [reply composer.json]: @daniel-rose Yes now

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**